docs/index.rst

```diff
@@ -28,6 +26,25 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDEs
+-------------
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
```

Comment:
Shouldn't this be the first as it is related to the Solidity project? Maybe a better (more meaningful) name could be used, e.g. *In-browser Solidity compiler and simulator*?